### DISCUSSION TITLE
Show multiple states of the same robot model in the same scene

### DISCUSSION THREAD

**Discussion by eirtech** [stack-of-tasks/pinocchio#1980] [Q&A] [answered]
Hi, 

First, I want to thank you for your great work.

I want to show the same robot model in the same scene but in different states. I am not sure how to make it with meshcatvisulizer. I already tried to make 2 model, collisionModel, etc. variables, for example 2 times to show 2 different state of the robot. but was not successful. Would you please help me in this regard? 

Thanks in advance.

**Comment by jcarpent** [comment, marked as answer]
Hi @eirtech ,

Here is a tiny example https://github.com/stack-of-tasks/pinocchio/blob/a8ba226fc99213b66c98bcbe425827a633cde41d/examples/meshcat-viewer.py#L65
